middleware/proxy/google.go

```diff
@@ -218,23 +219,30 @@ func newUpstream(hosts []string, old *staticUpstream) Upstream {
 			Conns:       0,
 			Fails:       0,
 			FailTimeout: upstream.FailTimeout,
-			Unhealthy:   false,
+			OkUntil:     time.Now().Add(time.Duration(24 * 365 * 100 * time.Hour)), // now + 100 years
```

Comment:
probably better as a constant. HundredYears or something better, but pondering if the zero value of time.Time{} wouldn't be better.